docs/components/button.md

```diff
@@ -148,6 +150,36 @@ KButton also supports the disabled attribute with both Button and Anchor types.
 <KButton to="http://google.com" appearance="btn-link" disabled>Disabled Native Anchor Link</KButton>
 ```
 
+:::warning NOTE
+Should you need to use a KTooltip component on a KButton with `disabled` attribute, don't forget to wrap an additional tag around your KButton, like shown in the example below. Otherwise KTooltip won't be triggered since elements with `disabled` attribute don't trigger pointer events.
```

Comment:
Is this an issue from the Kongponent level that we are trying to resolve by wrapping an element?